common/test/views/support/cleaner/AtomCleanerTest.scala

```diff
@@ -89,7 +89,7 @@ class AtomCleanerTest extends FlatSpec
   "Youtube template" should "include duration" in {
     val html = views.html.fragments.atoms.media(media = youTubeAtom.map(_.media.head).get, displayCaption = false, mediaWrapper = None)(TestRequest())
     val doc = Jsoup.parse(html.toString())
-    doc.getElementsByClass("youtube-media-atom__bottom-bar__duration").asScala.headOption should be(defined)
+    doc.getElementsByClass("youtube-media-atom__bottom-bar__duration").html() should be("0:36")
   }
 
 
```

Comment:
Would it make sense to test the `formattedDuration` function with the most common/edge cases?